src/main/java/interface_adapter/results/ResultsState.java

```diff
@@ -13,6 +13,7 @@ public class ResultsState {
     private List<University> universities = Collections.emptyList();
     private University chosenUniversity = null;
     private String searchError = null;
+    public String prevView = "";
```

Comment:
Good addition to the ResultsState! This allows other views to be instantiated to it without altering any code in previous views.